### PR TITLE
fix #39 merge create project howto

### LINKED ISSUE #39: Two pages about creating projects
One page is for creation from QGIS plugin, the other page from the web - but they don't have different names and both are in "How to" section:
- https://merginmaps.com/docs/howto/manage/plugin/plugin-new-project/
- https://merginmaps.com/docs/howto/manage/web/web-create-project/

### PR DESCRIPTION
**Pull request by alex-cit**
fix #39 
Created a new joint page in "How to" about creating new projects using all platforms (Input, QGIS, cloud) by merging and editing two existing pages (https://merginmaps.com/docs/howto/manage/web/web-create-project/ , https://merginmaps.com/docs/howto/manage/plugin/plugin-new-project/ ) and added instructions for Input from https://merginmaps.com/docs/tutorials/capturing-first-data/

### CODE REVIEW

File: src/howto/create-project/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+2. From the top right of the panel, select **Clone**
+   ![](./web-clone-project.png)
+3. In the new window, you can change the project name and select the owner
+<!-- add screenshot? -->
+
+
+
+
+
+
```
Comment (PeterPetrik) [inline src/howto/create-project/index.md:121]: - Please remove extra white lines (ideally you can create a single newline at the end of the file)
- remove commented out code `<!-- add screenshot? -->`, either add screenshot, or create a ticket, I doubt anyone will remember "TODOs" in the text :)

**Review by PeterPetrik — COMMENTED** [review]
Thanks, looks good. Broken links check is failing, I think you forgot to cleanup the index https://github.com/MerginMaps/docs/blob/main/src/index.md for the old links

File: src/howto/create-project/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Create a new project
+[[toc]]
+
+There are several methods of creating a new <MainPlatformName /> project:
+- [<MobileAppName /> ](./index.md#create-a-project-in-mergin-maps-input) offers the quickest (albeit limited) way of creating a <MainPlatformName /> project.
```
Comment (PeterPetrik) [inline src/howto/create-project/index.md:5]: can you please double-check in generated version that the `MobileAppName` expands when used as title inside the link?

Comment (alex-cit) [reply src/howto/create-project/index.md:5]: This seems to work.

File: src/howto/create-project/index.md (hunk 1 above)
Comment (alex-cit) [reply src/howto/create-project/index.md]: Cleaned up, the only 404 left is https://github.com/merginmaps/docs/edit/main/src/howto/create-project/index.md - the link from the "Help us improve this page!".

Comment (PeterPetrik) [reply src/howto/create-project/index.md]: created bug issue for this: https://github.com/MerginMaps/docs/issues/80

**Pull request merged**